src/qgis_ros/core/crs.py

```diff
@@ -12,5 +12,10 @@
                 +units=m \
                 +no_defs'
 
+PROJ4_WGS84 = '+proj=longlat +datum=WGS84 +no_defs'
 
 simpleCrs = QgsCoordinateReferenceSystem('PROJ4:' + PROJ4_SIMPLE)
+wgs84Crs = QgsCoordinateReferenceSystem('PROJ4:' + PROJ4_WGS84)
+
+proj4CrsDict = {'simple': PROJ4_SIMPLE, 'wgs84': PROJ4_WGS84}
+crsDict = {'simple': simpleCrs, 'wgs84': wgs84Crs}
```

Comment:
Needs return at end of file.

Reply:
To confirm, you mean a newline character.  Probably a windows vs. linux or code editor detail. I'll add it back in here.